src/components/auction/Auction.js

```diff
@@ -15,7 +15,7 @@ const Auction = props => {
     auctionStatus,
     buyDisabled,
     navigation,
-    title
+    title,
```

Comment:
Please check if Prettier is adding the trailing comma. Not adding it is my preference (and adds less diffs).

Reply:
prettier 2.0 does add a trailing comma. I did not update prettier to avoid adding it (although I would add it in a separated PR 'cause I think it works better for diffs when trailing comma is there).

But perhaps the linter has added it  - because it is only added in the files I had to update

Reply:
Please disable trailing commas to reduce the number of diffs.

Reply:
Removed here 13bc66e456f68d3c80b8d1d57dc5e3290ea1bf95